pkg/service/aws/kubec.go

```diff
@@ -50,15 +50,7 @@ func (ctrl AWSController) GetKubeConfig(ctx context.Context, req *proto.GetKubeC
 
 	authinfos := make(map[string]*clientcmdapi.AuthInfo)
 	authinfos[defaultCluster] = &clientcmdapi.AuthInfo{
-		Exec: &clientcmdapi.ExecConfig{
-			Command: "aws",
-			Args: []string{
-				"--region", region,
-				"eks", "get-token",
-				"--cluster-name", clusterName,
-			},
-			APIVersion: "client.authentication.k8s.io/v1alpha1",
-		},
+		Token: kubeConfig.BearerToken,
```

Comment:
this will change the bahaviour of the kubeconfig file generated using spawner cli too, do we want that happen?

Reply:
Yup

Reply:
token is very short lived, 15min

Reply:
True, that could be an issue. How about we pass a flat in `GetKubeConfig` request to get raw token?

Reply:
yeah, i think that would helpful. even if its 15min its very useful for our testing, adding flag for command would help use to allow testing different cluster.